examples/resources/pingfederate_server_settings/resource.tf

```diff
@@ -7,12 +7,60 @@ resource "pingfederate_server_settings" "serverSettingsExample" {
     phone      = "555-555-1222"
   }
 
-  federation_info = {
-    base_url = "https://localhost:9999"
-  }
+  notifications = {
+    license_events = {
+      email_address = "license-events-email@company.com"
+      notification_publisher_ref = {
+        id = "<uiInstanceID>"
+      }
+    }
+    certificate_expirations = {
+      email_address          = "cert-expire-notifications@company.com"
+      initial_warning_period = 45
+      final_warning_period   = 7
+      notification_publisher_ref = {
+        id = "<uiInstanceID>"
+      }
+    }
+    notify_admin_user_password_changes = true
+    account_changes_notification_publisher_ref = {
+      id = "<uiInstanceID>"
+    }
+    metadata_notification_settings = {
+      email_address = "metadata-notification@company.com"
+      notification_publisher_ref = {
+        id = "<uiInstanceID>"
+      }
+    }
+
+
+    federation_info = {
+      // base_url must be standard URL format: http(s)://<company-or-hostname> with optional domain and port
+      base_url = "https://localhost:9999"
+      // SAML entities have to be defined first
+      saml2_entity_id  = "urn:auth0:example:myserverconnection2"
```

Comment:
it looks like a few of the saml properties are off from the schema change in this file